.yamato/project-test.yml

```diff
@@ -132,4 +132,4 @@ develop_pull_request_trigger:
     pull_requests:
     - targets:
         only:
-          - "develop"
\ No newline at end of file
+          - "release/0.1.0"
```

Comment:
@JesseOlmer — I'm not sure if I should replace `develop` with `release/0.1.0` or add `release/0.1.0` additionally here.
I think this works for this branch because we'll leave this config here and not merge it back into `develop`, right?

Reply:
We will merge back to develop, so you should add release. Also, I'd suggest you add `"/release\/.+/"` and `"master"` so we don't run into this next time

Reply:
It's unfortunate that we'll have to remember to do this each release we cut.  Unless, are wildcards supported?

Reply:
I added what Jesse said `"/release\/.+/"` which basically mean we now have "release/*" wildcard :)